SerializedData.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using UnityEngine;
+
+namespace DUCK.PersistentDataStore
+{
+	/// <summary>
+	/// A persistent data store for serializable objects, identified by their type, or optionally by a unique ID.
+	/// </summary>
+	public static class SerializedData
+	{
+		/// <summary>
+		/// The data path used to save persistent objects.
+		/// </summary>
+		private static readonly string saveDataPath = Application.persistentDataPath + "/PersistentData/";
+
+		/// <summary>
+		/// Check if an object of type T exists in the persistent store, optionally with a unique ID.
+		/// <param name="uid">The unique ID (optional) - will check for the default object name if not specified.</param>
+		/// <returns>bool - whether or not the specified object exists.</returns>
+		/// </summary>
+		public static bool Exists<T>(string uid = "")
+		{
+			return File.Exists(GetFilePath<T>(uid));
```

Comment:
This is going to create me a directory and all I wanted was a boolean value.

Reply:
^